### ISSUE TITLE
Dynamite Headdy - Picodrive - 'Attract Mode' achievement

### ISSUE THREAD

**Issue by chashodges** [meleu/RetroArch-problematic-cheevos#17]
The following achievement doesn't seem to work on Retroarch at present - http://retroachievements.org/Achievement/13297. I've tried using Picodrive and Genesis Plus.

The achievement should trigger by pressing start at the title screen, and then letting the 90 second cutscene play out entirely.

**Comment by meleu** [comment]
I confirm it happens on RetroArch, but I didn't test on RAGens...

**Comment by leiradel** [comment]
I'm not getting cheevos for this game, maybe there's a problem with the hash?

```
[INFO] CHEEVOS system RAM: 000000000727E920 65536
[INFO] CHEEVOS save RAM:   0000000006CE1910 16384
[INFO] CHEEVOS video RAM:  0000000000000000 0
[INFO] CHEEVOS RTC:        0000000000000000 0
[INFO] CHEEVOS testing Genesis (6Mb padding).
[INFO] CHEEVOS getting game id for hash a8763564d9a18c588e1221d48808e0a2
[INFO] CHEEVOS got game id 0.
[INFO] CHEEVOS testing NES (discards VROM).
[INFO] CHEEVOS testing Generic (plain content).
[INFO] CHEEVOS getting game id for hash 2de73ba9ef04f9d1007c77cd818e634a
[INFO] CHEEVOS got game id 0.
[INFO] CHEEVOS this game doesn't feature achievements.
```

**Comment by meleu** [comment]
@leiradel I tried again and realized that it's a "false positive" bug. I started the game and was watching the demo and then my youngest kid woke up. I left the game in the demo and went to give attention to the kid. Later I came back to the computer and look at the logs:

```
[INFO] CHEEVOS system RAM: 0x7f01bb151740 65536
[INFO] CHEEVOS save RAM:   0x1d19790 16384
[INFO] CHEEVOS video RAM:  (nil) 0
[INFO] CHEEVOS RTC:        (nil) 0
[INFO] CHEEVOS testing Genesis (6Mb padding).
[INFO] CHEEVOS getting game id for hash ed9842ed0f59fc8a6385bfaac6900d0e
[INFO] CHEEVOS got game id 221.
[INFO] CHEEVOS got achievements for game id 221.
[INFO] CHEEVOS deactivated unlocked cheevo 13363 (Cleared Scene 1).
[INFO] CHEEVOS deactivated unlocked achievements in softcore mode.
[INFO] CHEEVOS deactivated unlocked cheevo 13363 (Cleared Scene 1).
[INFO] CHEEVOS deactivated unlocked achievements in hardcore mode.
[INFO] CHEEVOS posted playing activity.
[INFO] CHEEVOS awarding cheevo 13296: He's Dead, Jim (Die for the first time.).
[INFO] CHEEVOS awarded achievement 13296.
[INFO] CHEEVOS awarding cheevo 13297: Attract Mode (Watched the entire demo.).
[INFO] CHEEVOS awarded achievement 13297.
```

I earned a cheevo for the first die and after earned the cheevo for watching the demo. Therefore the demo you have to watch is the one before starting the game, not the one after like @chashodges reported.

@chashodges : please check it on your end. You have to watch the demo **before** pressing start.

Closing the issue.

**Comment by leiradel** [comment]
Great, but then **He's Dead, Jim** shouldn't be awarded, right? Can someone check the behavior in the official emulator, and open another issue if it's not awarded on it? Thanks.

**Comment by meleu** [comment]
The description of [He's Dead Jim](http://retroachievements.org/Achievement/13296) says "Die for the first time". I think everything is fine.

That little story about my morning was told to say:
1. I started the game and didn't touch any input, so I watched the entire demo **after starting the game** (press start on title screen), and didn't get the [Attract Mode](http://retroachievements.org/Achievement/13297) cheevo.
2. The actual game started and I was away from the computer, the character died and I got the **He's Dead, Jim** cheevo.
3. Then I suppose I lost the lives left and got a game over and the game went back to the title screen.
4. The **demo before pressing started** played entirely and I got the **Attract Mode** cheevo.

That's why I think everything is fine. Got it? :)
